package/windows/ReactNativeXaml/XamlMetadata.h

```diff
@@ -235,7 +235,8 @@ struct XamlMetadata : std::enable_shared_from_this<XamlMetadata> {
   void PopulateCommands(const winrt::Windows::Foundation::Collections::IVector<winrt::hstring>& commands) const;
 
   void JsiDispatchEvent(facebook::jsi::Runtime& rt, int64_t viewTag, std::string&& eventName, std::shared_ptr<facebook::jsi::Object>& eventData) const noexcept;
-  std::optional<facebook::jsi::Function> m_callFunctionReturnFlushedQueue;
+  std::optional<facebook::jsi::Function> m_receiveEvent;
+  std::optional<facebook::jsi::Object> m_eventEmitter;
```

Comment:
do we need to store the event emitter if we're only using m_receiveEvent?

Reply:
The method isn't bound so we use m_eventEmitter when we are calling the method using callWithThis.

Reply:
my bad, read wrong :-D